pkg/features/options.go

```diff
@@ -24,7 +24,7 @@ func withUnchangeable(unchangeable bool) option {
 }
 
 var (
-	enabled            = withEnabledByDefault() // Deprecated: just remove the flag.
+	released           = withReleased() // Deprecated: just remove the flag.
```

Comment:
Wait, do we really need this flag if it was already marked as deprecated (although not correctly)?

Reply:
* #12170